### PR TITLE
gravel: expose cephfs credentials via API

### PR DESCRIPTION
**Pull request by jecluis**
Resolves: #329 

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Comment by jecluis** [comment]
jenkins run tumbleweed

File: src/gravel/controllers/orch/cephfs.py [hunk 1]
```diff
@@ -82,3 +90,41 @@ def get_fs_info(self, name: str) -> CephFSListEntryModel:
...
+        except CephCommandError as e:
+            raise CephFSError(str(e)) from e
+        lst = parse_obj_as(List[CephFSAuthorizationModel], res)
+        assert len(lst) == 1
+        return lst[0]
+
+    def get_authorization(self, fsname: str) -> CephFSAuthorizationModel:
+        cmd = {
+            "prefix": "auth get",
+            "entity": f"client.{fsname}-default",
```
Comment (tserong) [inline src/gravel/controllers/orch/cephfs.py:118]: This hardcodes `-default`, but `authorize()` takes clientid as a parameter, so there's a slight mismatch here. Should `get_authorization()` also have clientid as a parameter?

**Review by tserong — APPROVED** [review]
LGTM, with one question

File: src/gravel/controllers/orch/cephfs.py (hunk 1 above)
Comment (jecluis) [reply src/gravel/controllers/orch/cephfs.py:118]: yeah, good call. Let me rework this a little bit.

**Comment by jecluis** [comment]
@tserong repushed, mind another look?

**Review by tserong — APPROVED** [review]

**Pull request merged**